source/common/json/config_schemas.cc

```diff
@@ -1210,6 +1210,10 @@ const std::string Json::Schema::CLUSTER_SCHEMA(R"EOF(
         "minimum" : 0,
         "exclusiveMinimum" : true
       },
+      "dns_lookup_family" : {
+        "type" : "string",
+        "enum" : ["v4_only", "v6_only", "auto"]
```

Comment:
The problem with auto is that it's a keyword and can't be used when making the string->enum conversion.